emission/core/wrapper/placeuserinput.py

```diff
@@ -5,27 +5,23 @@
 from future import standard_library
 standard_library.install_aliases()
 from builtins import *
-import logging
+import emission.core.wrapper.userinput as ecwui
 import emission.core.wrapper.wrapperbase as ecwb
 
-class Placeuserinput(ecwb.WrapperBase):
-    props = {"enter_ts": ecwb.WrapperBase.Access.RO,    # geojson representation of the point
-             "enter_local_dt": ecwb.WrapperBase.Access.RO, # start datetime in local time
-             "enter_fmt_time": ecwb.WrapperBase.Access.RO, # start formatted time (in timezone of point)
-             "exit_ts": ecwb.WrapperBase.Access.WORM, # end UTC timestamp (in secs)
-             "exit_local_dt": ecwb.WrapperBase.Access.RO, # end datetime in local time
-             "exit_fmt_time": ecwb.WrapperBase.Access.RO, # end formatted time (in timezone of point)
-             "label": ecwb.WrapperBase.Access.RO, # string summary of the survey
-             "version": ecwb.WrapperBase.Access.RO, # the survey version
-             "name": ecwb.WrapperBase.Access.RO, # the survey name
-             "xmlResponse": ecwb.WrapperBase.Access.RO, # the XML string representation of the survey response
-             "jsonDocResponse": ecwb.WrapperBase.Access.RO # the JSON representation of the survey response
-    }
+class Placeuserinput(ecwui.Userinput):
+    props = ecwui.Userinput.props
+    props.update({"start_ts": ecwb.WrapperBase.Access.RO,    # geojson representation of the point
+             "start_local_dt": ecwb.WrapperBase.Access.RO, # start datetime in local time
+             "start_fmt_time": ecwb.WrapperBase.Access.RO, # start formatted time (in timezone of point)
+             "end_ts": ecwb.WrapperBase.Access.WORM, # end UTC timestamp (in secs)
+             "end_local_dt": ecwb.WrapperBase.Access.RO, # end datetime in local time
+             "end_fmt_time": ecwb.WrapperBase.Access.RO, # end formatted time (in timezone of point)
+    })
 
     enums = {}
     geojson = []
     nullable = []
-    local_dates = ["enter_local_dt", "exit_local_dt"]
+    local_dates = ["start_local_dt", "end_local_dt"]
```

Comment:
why is the placeuserinput using `start/end` instead of `enter/exit`?

Reply:
To me, enter/exit doesn't make sense here (you don't enter and exit an input). It's also just easier to work with one set of terms for the purpose of input matching

But if it's important to keep that terminology, we'll revert this and refactor the input matching to account for it